.travis.yml

```diff
@@ -7,8 +7,8 @@ before_install:
   - npm install -g codecov
 script:
   - npx lerna bootstrap --no-ci
-  - node packages/lib/build_mod --source-dir packages/slave/lua/clusterio_lib --bump-patch --factorio-version 0.18 --output-dir temp/test/sharedMods
-  - curl -o temp/test/sharedMods/subspace_storage_1.99.1.zip -L https://github.com/clusterio/factorioClusterioMod/releases/download/1.99.1/subspace_storage_1.99.1.zip
+  - node packages/lib/build_mod --source-dir packages/slave/lua/clusterio_lib --output-dir temp/test/sharedMods
+  - curl -o temp/test/sharedMods/subspace_storage_1.99.8.zip -L https://github.com/clusterio/factorioClusterioMod/releases/download/1.99.6/subspace_storage_1.99.8.zip
```

Comment:
Thats a bit awkward. Shouldn't this be specified to always pull the latest or something?

Reply:
The Subspace Storage plugin should be moved to the mod repository and tested from there.